src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_ILocalFunctionStatement.cs

```diff
@@ -458,5 +471,85 @@ void M(int p)
 
             VerifyOperationTreeAndDiagnosticsForTest<LocalFunctionStatementSyntax>(source, expectedOperationTree, expectedDiagnostics);
         }
+
+        [CompilerTrait(CompilerFeature.IOperation)]
+        [Fact]
+        public void TestInvalidLocalFunction_ExpressionAndBlockBody()
+        {
+            string source = @"
+class C
+{
+    void M(int p)
+    {
+        /*<bind>*/object F() => new object(); { return new object(); }/*</bind>*/;
```

Comment:
>new object(); [](start = 55, length = 13)

Consider using different expressions so that we could easily tell them apart. Applies to other added tests